environment_setup/install_clion.sh

```diff
@@ -4,12 +4,56 @@ echo "================================================================"
 echo "Installing CLion"
 echo "================================================================"
 
-# Clion can be installed through snapcraft
-# https://snapcraft.io/clion
-sudo apt-get install snapcraft
-sudo snap install clion --classic
+clion_version="2019.2.2"
+usr_location="/usr/local/bin/clion"
```

Comment:
perhaps `clion_executable_path` might be a bit more descriptive?

Reply:
done